axlearn/audio/frontend.py

```diff
@@ -0,0 +1,127 @@
+# Copyright © 2023 Apple Inc.
+
+"""Audio frontends for feature extraction."""
+
+from typing import Callable, Dict, Optional, Sequence, Union
+
+import jax.numpy as jnp
+
+from axlearn.audio.frontend_utils import (
+    WindowType,
+    linear_to_log_mel_spectrogram,
+    linear_to_mel_weight_matrix,
+    magnitude_spectrogram,
+    next_power_of_2,
+    pre_emphasis,
+    sliding_window,
+    windowing,
+)
+from axlearn.common.base_layer import BaseLayer
+from axlearn.common.config import (
+    REQUIRED,
+    InstantiableConfig,
+    Required,
+    config_class,
+    maybe_instantiate,
+)
+from axlearn.common.module import Module
+from axlearn.common.utils import Tensor
+
+
+def scale_by_mean_std(
```

Comment:
Looks like this is not just scaling, but also shifting. Is `normalize_by_mean_std` a more accurate name?

Reply:
Sounds good, updated.